dask_sql/physical/rel/custom/create_model.py

```diff
@@ -130,6 +131,12 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
         wrap_fit = kwargs.pop("wrap_fit", None)
         fit_kwargs = kwargs.pop("fit_kwargs", {})
 
+        if wrap_predict is False and "dask" not in model_class:
```

Comment:
Maybe it should be this?

```suggestion
        if wrap_predict is False and "dask" not in model_class.lower():
```